ToolsTest/Models/VladyslavClassArticleTests/ArticleTestConstructor_xUnit.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    public class ArticleTestConstructor_xUnit
+    {
+        [Fact]
+        public void CtorArticle_IsType_User()
+        {
+            //Arrange
+            User _user = new User("Tommy");           
+            //Act
+            Article _article = new Article(_user);
+            //Assert            
+            Assert.IsType<User>(_article.Autor);
+        }
+        [Fact]
+        public void CtorArticle_HasTitleHasText_NotNull()
+        {
+            //Arrange
+            string _title = "Title";  
+            string _text = "Text";
+            //Act
+            Article _article = new Article(_title, _text);
+            //Assert
+            Assert.NotNull(_article);          
```

Comment:
Here the test is never be failed as `_article` will never be not null due to initialization (using `new`) at row #28.
Check the expected properties istead.
```suggestion
            Assert.NotNull(_article.Title);
            Assert.NotNull(_article.Text);
```